worlds/ufo50/night-manor/items.py

```diff
@@ -0,0 +1,135 @@
+    "Shovel": ItemInfo(33, IC.progression),
+    "Bronze Key": ItemInfo(34, IC.progression, "keys"),
+    "Gold Key": ItemInfo(35, IC.progression, "keys"),
+    "Motor": ItemInfo(36, IC.progression),
+    "Steel Key": ItemInfo(37, IC.progression, "keys"),
+    "Hacksaw": ItemInfo(38, IC.progression),
+    "Silver Key": ItemInfo(39, IC.progression, "keys"),
+    "Ring": ItemInfo(40, IC.progression),
+    "Brass Key": ItemInfo(41, IC.progression, "keys"),
+    "Gear": ItemInfo(42, IC.progression),
+    "Green Gemstone": ItemInfo(43, IC.progression, "gems"),
+    "Journal Entry 12": ItemInfo(44, IC.filler, "journal_entries"),
+    "Journal Entry 15": ItemInfo(45, IC.filler, "journal_entries"),
+    "Journal Entry 6": ItemInfo(46, IC.filler, "journal_entries"),
+    "Magnifying Glass": ItemInfo(47, IC.progression),
+    "Tea Tree Oil": ItemInfo(48, IC.progression),
+    "Hydrogen Peroxide": ItemInfo(49, IC.progression),
+    "Safe Combination": ItemInfo(50, IC.progression),
+    "Journal Entry 16": ItemInfo(51, IC.filler, "journal_entries"),
+    "Cigar Butt": ItemInfo(52, IC.progression),
+    "Yellow Gemstone": ItemInfo(53, IC.progression, "gems"),
+    "Journal Entry 13": ItemInfo(54, IC.filler, "journal_entries"),
+    "Computer Password": ItemInfo(55, IC.progression),
+    "Journal Entry 3": ItemInfo(56, IC.filler, "journal_entries"),
+    "Piano Wire": ItemInfo(57, IC.progression),
+    "Crossbow": ItemInfo(58, IC.progression),
+    "Doll": ItemInfo(59, IC.progression),
+    "Aluminum Key": ItemInfo(60, IC.progression, "keys"),
+    "Journal Entry 14": ItemInfo(61, IC.filler, "journal_entries"),
+    "Fungicide Recipe": ItemInfo(62, IC.progression),
+    "Fungicide": ItemInfo(63, IC.progression),
+    "White Gemstone": ItemInfo(64, IC.progression, "gems"),
+    "Glasses": ItemInfo(65, IC.progression),
+    "Maze Directions": ItemInfo(66, IC.progression),
+    "Crossbow Bolt": ItemInfo(67, IC.progression),
+    "Iron Key": ItemInfo(68, IC.progression, "keys"),
+    "Journal Entry 17": ItemInfo(69, IC.filler, "journal_entries")
+
+}
+
+night_manor_item_groups; Dict[str, Set[str]] = {
+    "Journal Entries": {"Night Manor - Journal Entry 1", 
+                        "Night Manor - Journal Entry 2",
+                        "Night Manor - Journal Entry 3",
+                        "Night Manor - Journal Entry 4", 
+                        "Night Manor - Journal Entry 5",
+                        "Night Manor - Journal Entry 6",
+                        "Night Manor - Journal Entry 7",
+                        "Night Manor - Journal Entry 8"
+                        "Night Manor - Journal Entry 9",
+                        "Night Manor - Journal Entry 10",
+                        "Night Manor - Journal Entry 11",
+                        "Night Manor - Journal Entry 12",
+                        "Night Manor - Journal Entry 13",
+                        "Night Manor - Journal Entry 14"
+                        "Night Manor - Journal Entry 15",
```

Comment:
```suggestion
                        "Night Manor - Journal Entry 7",
                        "Night Manor - Journal Entry 8",
                        "Night Manor - Journal Entry 9",
                        "Night Manor - Journal Entry 10",
                        "Night Manor - Journal Entry 11",
                        "Night Manor - Journal Entry 12",
                        "Night Manor - Journal Entry 13",
                        "Night Manor - Journal Entry 14",
                        "Night Manor - Journal Entry 15",
```
Python lets you concatenate strings by having them adjacent to each other.
`"test1" "test2" == "test1test2"`